docs/components/modeler/web-modeler/connectors/available-connectors/slack.md

```diff
@@ -0,0 +1,90 @@
+---
+id: slack
+title: Slack connector
+description: Working with Slack connector
+---
+
+The **Slack connector** allows you to interact with a [Slack](https://slack.com/) workspace, for example by sending Slack messages from your BPMN processes.
+
+## Prerequisites
+
+To use the **Slack connector** the following prerequisites need to be set-up. Read the [appendix](#appendix) to find out more.
+
+1. [Slack workspace](#use-a-well-known-slack-workspace) - this is the workspace the **Slack connector** will communicate with
+2. [Slack basic app with bot token configured](#configure-a-basic-slack-app) - the **Slack connector** will communicate through this Slack app with the workspace - you can consider the Slack app as _Slack bot representing the Camunda platform_
+3. [Slack bot token stored as secret](#store-slack-bot-token-as-secret) - the secret will store the Slack bot token and can be used to reference it from BPMN processes without revealing it in the BPMN `xml`
+
+## Create a Slack connector task
+
+To use a **Slack connector** in your process, follow the steps below:
```

Comment:
I'd like to keep this documentation focused and not detail how modeling works overall.

Instead the starting point could be: Transform a blank task to a `Slack: Send message task` _or_ append a `Slack: Send message task`.